university-agent/src/test/java/nl/quintor/studybits/controller/ScenarioIT.java

```diff
@@ -41,14 +48,24 @@
 
     static String schemaId;
 
+    static Issuer university;
+    private static void removeIndyClientDirectory() throws Exception {
```

Comment:
This function can be removed again, since the + System.currentTimeMillis() strategy provides local rerunnability.